front/src/Phaser/Game/GameScene.ts

```diff
@@ -61,11 +63,15 @@ export class GameScene extends Phaser.Scene implements GameSceneInterface{
             'resources/characters/pipoya/Male 01-1.png',
             { frameWidth: 32, frameHeight: 32 }
         );
+        this.load.bitmapFont('main_font', 'resources/fonts/arcade.png', 'resources/fonts/arcade.xml');
+
         cypressAsserter.preloadFinished();
     }
 
     //hook initialisation
-    init(){}
+    init(data: GameSceneInitDataInterface) {
```

Comment:
The player name should come from the GameManager class imo. Passing it as a scene parameter means we will have to pass it as parameter everytime. Also it is weird to have every user data come from GameManager EXCEPT the player name.